docs/installing-northstar/northstar-installers/viper-guide.md

```diff
@@ -0,0 +1,65 @@
+# Viper Guide
+
+Viper is a mod manager/installer for Northstar, a community made mod for Titanfall 2, that automatically updates mods and Northstar versions.
+
+Similar to FlightCore, Viper works on both Windows and Linux and have similar use cases, mostly differing in downloading Viper.
+
+## Windows Installation
+
+![The Windows download button on Viper's GitHub page](../../images/viper-windows-download.png)
+
+Install Viper from the [Northstar website](https://northstar.tf), the newest `.exe` file from [Viper's GitHub releases page](https://github.com/0neGal/viper/releases/), or the Windows download button on [Viper's main GitHub page](https://github.com/0neGal/viper).
+
+After this, run the file you downloaded to install and open Viper.
+
+## Linux Installation
+
+To install Viper on Linux, you want to download the newest `AppImage` file from the [Viper release page](https://github.com/0neGal/viper/releases), store it in a place that you want to, and make sure to right click on it, click on properties and tick `Executable as Program` on the AppImage file as shown below to allow it to be run on your computer. Double clicking the file after doing this will launch Viper.
```

Comment:
You forgot [the Flatpak](https://flathub.org/apps/com.github._0negal.Viper) >.<
(You shall be forgiven :P)